app/components/candidate_interface/new_reference_status_line_component.html.erb

```diff
@@ -0,0 +1,19 @@
+<% history.each do |event| %>
+  <p class="govuk-body-s govuk-!-margin-top-2 govuk-!-margin-bottom-0">
```

Comment:
This paragraph tag should only get included when there is a status line to display...